.vtexignore

```diff
@@ -7,5 +7,4 @@ react/**/node_modules/**
 react/**/__tests__/**
 react/**/__mocks__/**
 react/**/*.test.js
-react/testUtils/**/*
-react/.babelrc
\ No newline at end of file
+react/testUtils/**/*
```

Comment:
You also don't need this folder. Can delete it as well

Reply:
Fixed